.npmignore

```diff
@@ -1,4 +1,3 @@
 /example
-/__tests__
```

Comment:
Should be changed to just `__tests__` or better, let's use `files` entry in package.json to distribute only what we want

Reply:
I removed this line, as it's inside the `src/` now anyway.

Reply:
Ah yea, didn't notice we ignore src